pages/vi/vi-faq.md

```diff
@@ -134,6 +134,83 @@
     vagrant up
     ```
 
+#### Q18: What if I accidentally resign my manager account, or delete my ole--vagrant-vi folder, how do I recreate my manager profile?
+
++ You can follow these steps:
+
+	1. check the original folder where ole--vagrant-vi lies, if the folder still exists: run 
+	   
+	   ```
+	   cd ole--vagrant-vi
+	   	vagrant global-status
+		vagrant destroy [imagename]
+		cd ..
+		rm -rf ole--vagrant-vi
+		git clone https://github.com/dogi/ole--vagrant-vi.git
+		cd ole--vagrant-vi
+		vagrant up
+		```
+	if not, just deleted the image and recreate it. 
+	
+		```
+		vagrant global-status
+		vagrant destroy [imagename]
+		git clone https://github.com/dogi/ole--vagrant-vi.git
+		cd ole--vagrant-vi
+		vagrant up
+		```
+	2. You may run into error messege in creating virtual image using ```vagrant up ```:
+	 
+	 ```
+	 A Virtual Box machine with the name 'vi' already exists.
+Please use another name or delete the machine with the existing
+name, and try again.
+	 ```
+	 Try running```VBoxManage list vms && VBoxManage list runningvms```
```

Comment:
Should be a single sentence. No need to do newline here. It looks weird if the screen size changes (E.g. mobile devices)